### DISCUSSION TITLE
Hot reload does not update the DOM

### DISCUSSION THREAD

**Discussion by reitbauer** [quasarframework/quasar#9104] [CLI - BEX mode] [answered]
I have an issue with the quasar v2 beta 14.

The hot reload should work. I get the following in the browser console:
<img width="554" alt="image" src="https://user-images.githubusercontent.com/31218726/116601735-1a02a180-a92b-11eb-9c4f-349b4b1f75e2.png">

But the DOM does not reload.
For example: I change the title in the toolbar of the Mainlayout.vue but in the browser the DOM does not change.

I use:
- Mac OS Big Sur
- Visual Studio Code
- quasar v2 Beta 14
- node 14.16.0
- npm 7.11.1
- command: quasar dev
- Google Chrome 90.0.4430.93

Thanks in advance

**Comment by rstoenescu** [comment, marked as answer]
Hi,

The HMR (Hot Module Reload) is handled by vue-loader. If there's any issue that you encounter with it, please report it to [vue-loader](https://github.com/vuejs/vue-loader). Mention it's vue-loader v16.2.0. Thanks.

**Comment by DaveDisco** [comment]
I'm having a hot reload problem too. All seemed ok but I've come across two .vue files that have a problem. The first change you make is ok, it does the reload but after that nothing, no further changes trigger a reload. The only way to carry on is to restart quasar.  

rstoenescu...  I looked to report this on vue-loader but the latest version is v16.1.2 is this what quasar is using?

**Reply by rstoenescu** [reply]
v16.1.2 is not the latest. 16.2.0 is.
https://github.com/quasarframework/quasar/blob/vue3-work/app/package.json#L107

https://github.com/vuejs/vue-loader/blob/next/package.json#L3
